### DISCUSSION TITLE
Embed audio in docs

### DISCUSSION THREAD

**Discussion by irzadi** [mkdocs/mkdocs#2505] [Q&A]
Hey guys,
I've just started writing some lessons in mkdocs for my students who are currently in quarantine. I'm teaching languages so I'd love to be able to embed some audio inside the docs. Best if the students can click play then immediately hear the audio without having to leave the page".

Wiktionary and many dictionary sites do this already.
What is the best way of doing this in mkdocs ?

**Comment by oprypin** [comment]
You can find out how to insert [audio using HTML](https://www.google.com/search?q=audio+using+HTML "Google search") and then just write that HTML in the middle of your Markdown document.

**Comment by rmscode** [comment]
I'm late to the party, but [mkdocs-audio](https://github.com/jfcmontmorency/mkdocs-audio) allows you to embed audios on the documentation pages using a simple Markdown syntax.

You can install it via `pip` but be weary . . . As of writing this, there is a bug in this version where `</br>` line break tags aren't rendered in the final HTML. 

```
pip install mkdocs-audio
```

I did contribute to the project and fix the bug, so if line breaks are something you think you'll use a lot, download and install it from the github repo.

```
cd mkdocs-audio
python setup.py install
```
